middlewares/insertNonce.js

```diff
@@ -24,22 +26,29 @@ module.exports = (app, config) => {
   let html;
   let index_file = config.siteIndexFile;
 
+  if (index_file === 'undefined' || typeof index_file !== 'string') {
+    console.log("Salus: skipping nonce bc index file in salus.config.js not set")
```

Comment:
Not sure about this too

Reply:
The second is enough, if it is === undefined it is also !== string.

Reply:
good point, fixed as well